### ISSUE TITLE
NTR: 2-hydroxyphenylacetic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8574]
New term request 2-hydroxyphenylacetic acid high in urine (segal_030509144123):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000274
- label: 2--hydroxyphenylacetic aciduria
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: 
- def: Increased concentration of 2-hydroxyphenylacetic acid in the urine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8574
- synonyms:
